docs/sources/ingress.md

```diff
@@ -46,3 +46,6 @@ the values from that.
 
 2. Otherwise, iterates over the Ingress's `status.loadBalancer.ingress`, 
 adding each non-empty `ip` and `hostname`. 
+
+In the case that `--resolve-ingress-target-hostname` set, it will resolve hostnames in the status to create 
+A/AAAA records instead of CNAME.
```

Comment:
I would say this is a feature request to the controller that creates the status field.